Kuroba/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="pin_posts_not_watching_color">#898989</color>
```

Comment:
I'll fix this myself in the merge, but you don't really need to add `_color` when you'll be referring to the color by `R.color.<color>` in code anyways and that code isn't foldable in the editor so you'll always know it's a color. Small readability thing.